protos/sources/vega/vega.proto

```diff
@@ -1242,6 +1242,10 @@ message PerpetualData {
   int64 next_internal_composite_price_calc = 8;
   // The method used for calculating the internal composite price, for perpetual markets only.
   CompositePriceType internal_composite_price_type = 9;
+  // The last seen value of the settlement oracle.
+  string underlying_index_price = 10;
```

Comment:
To keep the terminology consistent this field should just be called `external_price`. Throughout all over the perpsy API we just use the terms internal/external.

Reply:
I don't have a preference but this name is what the ticket has https://github.com/vegaprotocol/vega/issues/10502#issuecomment-1916645422